Linked list with void pointers.c

```diff
@@ -39,7 +32,7 @@ void display_float(node *tail)
 }

```

Comment:
write code for display of chars, strings
write code for getting size of different data types
create 2 function pointers for display and get size